TODO

```diff
@@ -32,6 +32,10 @@ Janitorial Clean-ups:
 
 Features:
 
+* add a percentage syntax for TimeoutStopSec=, e.g. TimeoutStopSec=150%, and
+  then use that for the setting used in user@.service. It should be understood
+  relative to the configured default value.
```

Comment:
are you referring to the difference between DefaultTimeoutStopSec= value, and the default for the DefaultTimeoutStopSec= value? of course i mean, relative to the actual DefaultTimeoutStopSec= value in effect, which means 90s if nothing is configured there.

i.e. in the stop timeout hierarchy of defaults we have: TimeoutStopSec= → DefaultTimeoutStopSec= → 90s, and I think the relative setting in the first level should be taken towards the setting of the second level, if you follow what i try to say. It should not skip this second level, and be taken relative to the third level directly...